<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1629885834513' stop='1629885837633'>
    <name>upload new dataset on myDatasets page..</name>
    <title>upload new dataset on myDatasets page..</title>
    <test-cases>
        <test-case start='1629885834513' status='failed' stop='1629885837631'>
            <name>clear old e2e dataset</name>
            <title>clear old e2e dataset</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='dbd1fca8-ec2a-4e1d-9c69-57e2f5d125b1-attachment.png' type='image/png' size='51518'/>
            </attachments>
            <failure>
                <message>Expected 1 to equal 0.</message>
                <stack-trace>Error: Failed expectation
                    at Object.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\clear-old-e2e-projects-datasets-spec.ts:41:59)
                    at Generator.next (&lt;anonymous>)
                    at fulfilled (E:\vmware\Loop\GitHub\e2e\node_modules\tslib\tslib.js:112:62)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>